### ISSUE TITLE
ECL extension- causing programs not to run 

### ISSUE THREAD

**Issue by aldousveronica** [hpcc-systems/vscode-ecl#199]
There is an issue with the ECL extension. When I press the "run" button, VS Code acts like it's trying to run the program (you can see the green bar moving in the upper left corner of VS Code) but nothing actually happens, and eventually it just gives up trying to run. 

This issue can be solved by downgrading the extension's version. There have been a lot of recent updates to the ECL Language extension in VS Code. When I downgrade the extension version to one before all the updates, it works even with the newest version of VS Code.
 
Here is how to downgrade the ECL Language extension's version. This solves the problem where pressing the run button doesn't actually run your code.
 
1. Go to extensions tab
2. Click the gear on the ECL Language extension
3. Click install another version
4. Click 1.9.5
5. Reload vs code (I clicked the blue reload required button)
6. Try running your code again

**Comment by GordonSmith** [comment]
@aldousveronica thanks for your report - while I was unable to reproduce your issue, it is worth noting that this area is what changed the most in the v2.x release.  I did also notice that in some environments the submission process is quite slow and there was a definite lack of feed back.
It is also worth noting that submitting ECL via the Run/Debug pane is now deprecated and preferred method is via the new buttons at the top of the ECL Editor.
Selection of the launch configuration / target cluster can be done via the status bar.

1.  I have added an additional warning message when the user submits via the "old" mechanism.
2.  I have added additional progress information to fill the gap between clicking the submit button and when the WU shows up in the new Workunit History Tree.
3.  I have updated the [README.md](https://github.com/hpcc-systems/vscode-ecl) and [Marketplace](https://marketplace.visualstudio.com/items?itemName=hpcc-systems.ecl) with an improved graphic highlighting this:

![QuickStart](https://user-images.githubusercontent.com/966863/93851457-11b4fa80-fca8-11ea-9351-f5d6aa544fb5.gif)

Let me know if this resolves your issue?  If not we can do a screenshare later today?
